class DataIntegrityError(ValueError):
    """
    Вызывается, когда нарушаются ограничения целостности базы данных.
    """
